portsyncd/linksync.cpp

```diff
@@ -50,43 +49,6 @@ LinkSync::LinkSync(DBConnector *appl_db, DBConnector *state_db) :
     {
         string key = idx_p->if_name;
 
-        /* Explicitly store management ports oper status into the state database.
```

Comment:
Why do we need to keep the` for loop` as this was also added as part of #630

Reply:
Yes, we don't need this, removed to match initial PR changes